wiki/faq.md

```diff
@@ -642,4 +642,4 @@ have a lot of small transactions.
 7. Right-click the Gridcoin Shortcut, select properties, and remove `-choosedatadir`
 
 
----
\ No newline at end of file
+---

```

Comment:
what is going on down here?

Reply:
Hmm weird I don't remember adding that. Removed.